client/disputes/redirect-to-transaction-details/index.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { getHistory } from '@woocommerce/navigation';
+import { Spinner, Icon, Flex, FlexItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies.
+ */
+import Page from 'components/page';
+import { useDispute } from 'data/index';
+import { Charge } from 'wcpay/types/charges';
+import { getAdminUrl } from 'wcpay/utils';
+
+import './style.scss';
+import warning from 'wcpay/components/icons/warning';
+
+const RedirectToTransactionDetails: React.FC< { query: { id: string } } > = ( {
+	query: { id: disputeId },
+} ) => {
+	const { dispute, error, isLoading } = useDispute( disputeId );
+
+	useEffect( () => {
```

Comment:
Doh! Is this the fix for my react issue! Obvious in hindsight that I would need to redirect as a side effect. Thanks for sorting this @Jinksi , major oversight on my side.